preferences/windows/base/kustomization.yaml

```diff
@@ -12,6 +12,7 @@ components:
   - ../../components/diskbus-sata
   - ../../components/interfacemodel-e1000e
   - ../../components/tablet-usb
+  # Setting 1 hour termination grace period, because Windows can take longer to shut down.
```

Comment:
```suggestion
  # Setting termination grace period of 1 hour (3600 secs), because Windows can take longer to shutdown or reboot when updates or other changes are applied.
```

Reply:
done